### ISSUE TITLE
Annotation for field name

### ISSUE THREAD

**Issue by kaendfinger** [mezoni/binary_marshalling#1]
Consider the class:

```dart
class Group {
  String gr_name;
  String gr_passwd;
  int gr_gid;
  List<String> gr_mem;
}
```

This works, however I want to provide a very friendly API while still using binary marshalling. Removing the gr_ prefixes would make APIs easier to understand.

I propose an annotation that I can use on my class fields which will specify a name for the field. Such as `@StructName("gr_name")`.

**Comment by mezoni** [comment]
I know about.
Only need good name for annotation class.
Please, provide more options.

- StructName
- StructField
- Member
- Alias
- ...

**Comment by mezoni** [comment]
- BindTo
- Field

Better way is to use names similar to used in the database annotations.

**Comment by kaendfinger** [comment]
`@NativeName` sounds good.

**Comment by mezoni** [comment]
@kaendfinger 

>> @NativeName sounds good.

OK

**Comment by mezoni** [comment]
Available in version 0.0.3.
Added possibility to specify a name for the binary field in the class member declaration through the annotation `NativeName`.

**Comment by kaendfinger** [comment]
Thanks :)
